test/integration/gcp/controls/forseti.rb

```diff
@@ -0,0 +1,136 @@
+# encoding: utf-8
+# copyright: 2018, The Authors
```

Comment:
Drop this comment, since the following boilerplate specifies the copyright date and copyright owner.